.github/workflows/tests.yml

```
@@ -106,6 +107,11 @@ jobs:
         if: matrix.target == 'plugins'
         run: bin/rake plugin:pull_compatible_all
 
+      - name: Add hosts to /etc/hosts, otherwise Chrome cannot reach minio
+        run: |
+          sudo echo "127.0.0.1 minio.local" | sudo tee -a /etc/hosts
+          sudo echo "127.0.0.1 discoursetest.minio.local" | sudo tee -a /etc/hosts
```

🏌️ ("127.0.0.1" \*with emphasis\* 😅)

```suggestion
          echo "127.0.0.1 minio.local" | sudo tee -a /etc/hosts
          echo "127.0.0.1 discoursetest.minio.local" | sudo tee -a /etc/hosts
```